erpnext/agriculture/doctype/harvest/harvest.js

```diff
@@ -2,7 +2,21 @@
 // For license information, please see license.txt
 
 frappe.ui.form.on('Harvest', {
-	// refresh: function(frm) {
-
-	// }
+	setup: function (frm) {
+		frm.make_methods = {
+			'Stock Entry': () => {
+				if (frm.doc.docstatus === 1) {
```

Comment:
What happens for Draft documents?

Reply:
nope. It will not allowed to create stock for draft